### PR TITLE
Fix ToS link to point to correct file

### PR DESCRIPTION
**Pull request by caesar2164**
the external adminguide file moved, this updates link

@stvstnfrd & @mondiaz

### CODE REVIEW

**Comment by mondiaz** [comment]
@caesar2164 I see this is for suclass-theme. We don't have TOS on SUClass (see https://suclass.stanford.edu/tos) but we do have TOS on CME now (see https://cme.class.stanford.edu/tos).

**Comment by caesar2164** [comment]
@mondiaz CME uses the suclass theme...

File: templates/static_templates/theme-tos.html [hunk 1]
```diff
@@ -477,7 +477,8 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
             servicemarks appearing on the Site are the marks of their respective owners. Information about Stanford
             trademarks may be found at {trademark_link}.
         ''').format(
-            trademark_link='<a href="http://adminguide.stanford.edu/15_5.pdf">http://adminguide.stanford.edu/15_5.pdf</a>'
+            trademark_link='<a href="https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4">
+                https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4</a>'
```
Comment (stvstnfrd) [inline templates/static_templates/theme-tos.html]: Assuming you meant to split this into 2 lines, you should close the quotes on each line and rely on PYthon string concatenation. I don't think this passes pep8/pylint, either.

What about:
```python
trademark_link=(
    '<a href="https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4">'
    'https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4</a>'
)
```

File: templates/static_templates/theme-tos.html [hunk 2]
```diff
@@ -477,8 +477,10 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
             servicemarks appearing on the Site are the marks of their respective owners. Information about Stanford
             trademarks may be found at {trademark_link}.
         ''').format(
-            trademark_link='<a href="https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4">
-                https://adminguide.stanford.edu/chapter-1/subchapter-5/policy-1-5-4</a>'
+            trademark_link = (
```
Comment (stvstnfrd) [inline templates/static_templates/theme-tos.html]: `kwargs` should never have surrounding spaces, per pep8/pylint.

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**